tools/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+# Create a temp ssh key that will be used to login to the VMs
+SSH_KEY_DIR=$(mktemp -d)
+ssh-keygen -f "$SSH_KEY_DIR/key" -N "" -t ed25519
+
+# substitute SSH_KEY in any template
+for tpl in httpdir/*.tpl; do
+	dst="${tpl%.*}"
+	sed "s#%SSH_KEY%#$(cat "$SSH_KEY_DIR/key.pub")#" "$tpl" > "$dst"
+done
+
+# build with this key
+packer build -var "SSH_PRIVATE_KEY_FILE=$SSH_KEY_DIR/key" \
+    -var "SSH_PUB_KEY=$(cat "$SSH_KEY_DIR/key.pub")" \
+    "$1"
+
+rm -rf "$SSH_KEY_DIR"
+
+# convert to OVA
+VM_NAME=$(jq -r ".builders[].vm_name" < "$1")
```

Comment:
It would help a one-line comment with an example (e.g. `centos-7.json`) or assignment to an environment variable with a meaningful name to understand what `$1` is.